R/is_lint_level.R

```diff
@@ -0,0 +1,21 @@
+#' Is this an expression- or a file-level source object?
+#'
+#' Helper for determining whether the current `source_expression` contains
+#'   the full file tree, or is just a single expression.
```

Comment:
Full file tree isn't really clear to me. Maybe write "contains all expressions in the current file, or just a single expression." instead?